frontend/src/views/SecretMainPage/components/SecretListView/SecretListView.utils.ts

```diff
@@ -20,6 +20,15 @@ export const formSchema = z.object({
   overrideAction: z.string().trim().optional(),
   comment: z.string().trim().optional(),
   skipMultilineEncoding: z.boolean().optional(),
+
```

Comment:
Not sure how this is happening there seems to be some weird spacing between object keys etc.

Reply:
Probably a linting rule, but I think 1 space is good, it allows for separation. More than 1 space is not ideal though, which is how it is right now I believe